docs/jobs.md

```diff
@@ -143,15 +143,35 @@ Finished    Failed   Canceled
 * *Failed* - The task has failed. The error can be shown by ``hq job <job-id>``.
 * *Canceled* -  The task has been canceled by a user.
 
+## Time management
 
-## Time limit
+Time management is split into two settings:
+
+* **time limit** - It is maximal running time of a task.
+               If it is reached then task is terminated and set into FAILED state.
+               This setting does not have any impact on scheduling.
+* **time request** - The minimal remaining lifetime of a worker needed to start the task.
+                     The settings has impact only on the scheduling,
+                     When task is started, it has no effect
+                     if the task takes more/less computational time.
```

Comment:
```suggestion
* **time limit** - It is the maximal running time of a task.
               If it is reached then task is terminated and set into FAILED state.
               This setting does not have any impact on scheduling.
* **time request** - The minimal remaining lifetime of a worker needed to start the task.
                     Workers that do not have enough remaining lifetime will not be scheduled to run this task.
                     Once the task is scheduled and starts executing, it does not matter w.r.t this setting for how long it will actually run.
```